app/src/main/res/raw/menu.json

```diff
@@ -688,6 +688,56 @@
 					}
 				]
 			}
+		},
+		{
+			"version": 2018070101,
+			"url": "https://osem.aius.u-strasbg.fr/conferences/rmll2018/schedule.xml",
+			"title": "Rencontres Mondiales du Logiciel Libre 2018",
+			"start": "2018-07-07",
+			"end": "2018-07-11",
+			"metadata": {
+				"icon": "https://mamot.fr/system/accounts/avatars/000/070/390/original/e9f3cfd8460df6b1.png",
+				"links": [
+					{
+						"url": "https://2018.rmll.info/",
+						"title": "Website"
+					},
+					{
+						"url": "https://2018.rmll.info/fr/program/village",
+						"title": "Village du Libre"
+					}
+				],
+				"rooms": [
+					{
+						"name": "A[tT][rR][iI][uU][mM].*",
```

Comment:
Ah oh dear, what kind of inconsistency in the schedule is that? :-( I could just make the regex match case insensitive, in fact I'm surprised I haven't already..